### DISCUSSION TITLE
Wrap ruleset's icons if there are too many

### DISCUSSION THREAD

**Discussion by Frn1** [ppy/osu#24077] [Ideas]
Basically, if there are too many rulesets installed the default animation that plays when you open the game looks kinda weird because the icons go offscreen and but you are still able to see the borders. I know you see it for literally less than a second but it still bothers me. I don't know how hard this would be, but maybe they could wrap to a "next line" or something like text does when it doesn't fit.

**Comment by Joehuu** [comment]
Will be resolved in https://github.com/ppy/osu/pull/24078.

**Reply by Joehuu** [reply]
Reopening as proposed solution wasn't right. See https://github.com/ppy/osu/pull/24078#issuecomment-1885746950 and above for reasoning.
